### DISCUSSION TITLE
💖 looking for molecule docker driver maintainers

### DISCUSSION THREAD

**Discussion by ssbarnea** [ansible-community/molecule-docker#85] [Show and tell]
I am unable to actively maintain this driver as much as I would like, so I am looking for to add new maintainers. 

I can keep providing support for CI and integration with molecule itself but less around driver specifics.

**Comment by zhan9san** [comment]
@ssbarnea 

I want to handle it if you don't mind.

**Reply by ssbarnea** [reply]
Not at all! Ping me when you need help.
